### PR TITLE
basic error page styling for 404s and 500s

### PR DESCRIPTION
**Pull request by arches**
## Changelog
- removed static error pages
- routed 404s and 500s to controller actions
- made erb pages for 404s and 500s

## Link to issue:  
Fixes issue #267

## Steps for QA/Special Notes:
- You might need to set `config.consider_all_requests_local = false` to see these locally

## Relevant Screenshots: 
- super important for UI tasks!

## Are you ready for review?:

- [ ] Added relevant tests
- [ ] Linked PR to the issue
- [ ] Added notes for QA/special notes
- [ ] Added revelant screenshots

### CODE REVIEW

File: app/views/errors/internal_server_error.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<h1>Server Error</h1>
+
+<br>
+<%= image_tag "oops.gif", width: "400" %>
+
+<br>
+<br>
+
+<p>
+Something went wrong, please reach out to <%= mail_to "lockbox@midwestaccesscoalition.org" %>
```
Comment (BintLopez) [inline app/views/errors/internal_server_error.html.erb]: let's use the env var for this ... `ENV['LOCKBOX_EMAIL']`

File: app/views/errors/not_found.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<h1>Page Not Found</h1>
+
+<br>
+<%= image_tag "urkel.gif", width: "400" %>
+
+<br>
+<br>
+
+<p>
+Something went wrong, please reach out to <%= mail_to "lockbox@midwestaccesscoalition.org" %>
```
Comment (BintLopez) [inline app/views/errors/not_found.html.erb]: ditto re env var

File: app/controllers/errors_controller.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Okay, this class gets invoked for errors, by name,
+# if the request is not local (config.consider_all_requests_local).
+# This is because we configure `config.exceptions_app = routes`
+# in config/application.rb
+# For local requests (eg test and dev) it will instead go to some
+# fancy error renderer.
```
Comment (BintLopez) [inline app/controllers/errors_controller.rb]: this is a really helpful comment!

Comment (arches) [reply app/controllers/errors_controller.rb]: thanks, I copied this whole thing from work 🙃

**Pull request merged**